src/JsSse.php

```diff
@@ -190,8 +190,9 @@ protected function initSse(): void
         }
 
         // prevent buffering
-        if (ob_get_level()) {
+        while (ob_get_level() > 0) {
             ob_end_flush();
```

Comment:
please make all tests passing

Reply:
> please make all tests passing

Sorry, but I can't do this

Reply:
Obviously, we cannot merge anything that would break the whole CI :)

Might I know your usecase?

Reply:
I use the framework https://github.com/f3-factory/fatfree-core And SSE stops working in it.
Or are you talking about something else?

Reply:
The fix seems legit. Upper layer/framework can open more than 1 output buffer. I even wonder why the atk4 tests passed when they rely on ob. In anycase, now the tests needs to be fixed to pass.

Reply:
Unfortunately, I'm not good at testing.

Reply:
see https://github.com/atk4/ui/pull/2175#discussion_r1535566245